src/constants.ts

```diff
@@ -27,6 +27,9 @@ export const headerNames = {
   rootTransactionId: 'X-AR-IO-Root-Transaction-Id',
   dataItemDataOffset: 'X-AR-IO-Data-Item-Data-Offset',
   arnsTtlSeconds: 'X-ArNS-TTL-Seconds',
+  arnsName: 'X-ArNS-Name',
+  arnsBasename: 'X-ArNS-Basename',
+  arnsUndername: 'X-ArNS-Undername',
```

Comment:
unrelated to the PR, these are headers we want to add to exposed headers in cors config?

Reply:
cors dynamically supports all of the `values` in this constant https://github.com/ar-io/ar-io-node/blob/develop/src/app.ts#L58